server/src/instant/util/aws.clj

```diff
@@ -0,0 +1,48 @@
+(ns instant.util.aws
+  (:require [clj-http.client :as clj-http]
+            [amazonica.aws.ec2 :as ec2]))
+
+(def environment-tag-name "elasticbeanstalk:environment-name")
+
+(defn get-tag [tag-name]
+  (let [token (-> (clj-http/put
+                   "http://169.254.169.254/latest/api/token"
```

Comment:
curious q: where does this ip come from?

Reply:
This is the ip that aws (and the rest of the cloud services) uses for their metadata API: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

I'll add a comment.